volume-providers/vpc/provider/util.go

```diff
@@ -86,6 +90,34 @@ func skipRetry(err *models.Error) bool {
 	return false
 }
 
+// skipRetryForIKS skip retry as per listed error codes
+func skipRetryForIKS(err *models.IksError) bool {
```

Comment:
We can have only `skipRetry` and remove this method. As both methods do same thing. The input passed is different thats it.

Reply:
error type is different for both the methods, hence its required